skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
+    """Apply hysteresis thresholding to `image`.
+
+    This algorithm finds regions where `image` is greater than `high`
+    OR `image` is greater than `low` *and* that region is connected to
+    a region greater than `high`.
+
+    Parameters
+    ----------
+    image : array
```

Comment:
I think grayscale should be specified?

    image : (N, M[, ..., P]) ndarray
        Grayscale input image.